docs/crc-guide.md

```diff
@@ -0,0 +1,162 @@
+# Running Clowder on CRC 
+
+## Prerequisites
+-----------------
+1. Download the [crc binary](https://developers.redhat.com/products/codeready-containers/overview) and follow the instructions to get crc running.
+2. Fork or clone the [Clowder repo](https://github.com/RedHatInsights/clowder)
+3. Install [the Clowder dependencies](https://github.com/RedHatInsights/clowder#dependencies)
+4. Run `make install`
+
+
+## Running Clowder
+-------------------
+At this time it is not recommended to deploy Clowder to your crc cluster. Instead, we will run the Operator on your local machine. To do this, we need add `ingress-env-minio.clowder-system.svc` to your `/etc/hosts` localhost (127.0.0.1). 
+
+We're going to use Ingress as the example, so the configuration we're doing is specific to that. If you are standing up a different application, substitute your own services, or other variables. 
+
+`make run 2>&1 | grep '^{' | jq -r .`
+
+This will start the operator on your local machine with output redirected to `jq`. The jq output is formatted and easier to read and therefore recommended. However, if don't want that, `make run` will do just fine; albeit less neat. 
+
+## Applying the ClowdEnvironment
+---------------------------------
+Now that Clowder is running, we need to give it a `ClowdEnvironment` for Ingress to run inside. 
+
+In a new terminal, run `oc new-project clowder-system`
+
+Create the following as `clowd-environment.yaml`
+
+``` yaml
+---
+apiVersion: cloud.redhat.com/v1alpha1
+kind: ClowdEnvironment
+metadata:
+  name: ingress-env
+spec:
+  namespace: clowder-system
+  web:
+    port: 8000
+    provider: operator
+  metrics:
+    port: 9000
+    provider: operator
+    path: "/metrics"
+  kafka:
+    namespace: default
+    clusterName: crc-cluster
+    provider: local
+  db:
+    image: "registry.redhat.io/rhel8/postgresql-12:1-36"
+    provider: local
+  logging:
+    provider: none
+  objectStore:
+    provider: minio
+    port: 9000
+  inMemoryDb:
+    provider: redis
+  resourceDefaults:
+    limits: 
+      cpu: "500m"
+      memory: "8192Mi"
+    requests:
+      cpu: "300m"
+      memory: "1024Mi"
+```
+
+and then run `oc apply -f clowd-environment.yaml`
+
+Once applied, check the terminal that is running the operator and make sure there aren't any errors. If you're unsure, you can check the `clowder-system` namespace.
+
+Before we add the ClowdApp, we need to port forward the minio port on your local machine with `oc port-forward svc/ingress-env-minio 9000`
```

Comment:
A little context would help people understand why here. Just something simple like, because the operator needs to talk to Minio to create the buckets and Minio is running inside the cluster.....you know what I mean